Nasal/light-manager.nas

```diff
@@ -0,0 +1,303 @@
+# provides relative vectors from eye-point to aircraft lights
+# in east/north/up coordinates the renderer uses
+
+var light_manager = {
+    run: 0,
+
+    lat_to_m: 110952.0,
+    lon_to_m: 0.0,
+
+    light1_xpos: 0.0,
```

Comment:
There are lots of fields here getting a 0.0 value, but in the `init()` function they're initialized _again_. Can you try and see if just initializing the fields in the `init()` function is sufficient?